lib/date_extractors/heic_video_extractor.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:io';
+import 'package:gpth/date_extractors/json_extractor.dart';
+import "package:path/path.dart" as p;
+
+/// Finds corresponding json file with info and gets 'photoTakenTime' from it
+Future<DateTime?> heicVideoExtractor(File file, {bool tryhard = false}) async {
+  final fileNameWithoutExtension = p.basenameWithoutExtension(file.path);
+
+  if (fileNameWithoutExtension.startsWith("IMG") &&
+      p.extension(file.path).toLowerCase() == ".mp4") {
```

Comment:
Photo from iPhone with Live Photo enabled when uploading to Google Photos will create three files:
- `IMG_1845.HEIC`
- `IMG_1845.MP4`
- `IMG_1845.HEIC.json`

The `.MP4` file always comes in a pair with the `.HEIC` file when the live photo is enabled, however, there is no JSON file for the video. We can also use the JSON file for the image to set the date for the video. This will ensure that the image and the video are stored in the same directory.

Reply:
I noticed that it doesn't always start with IMG, for example, one of mine is like this:

```
97E2514D-F5BB-4314-B099-9A5FAE5EEA22.JPG
97E2514D-F5BB-4314-B099-9A5FAE5EEA22.JPG.json
97E2514D-F5BB-4314-B099-9A5FAE5EEA22.MP4
```